echo-notifications/src/main/groovy/com/netflix/spinnaker/echo/notification/DryRunNotificationAgent.java

```diff
@@ -59,7 +60,7 @@ public void sendNotifications(
     }
     log.info("Received dry run notification for {}", pipelineConfigId);
     Optional<Pipeline> match =
-        front50.getPipelines(application).stream()
+        AuthenticatedRequest.allowAnonymous(() -> front50.getPipelines(application)).stream()
```

Comment:
I'm not sure I follow how does this end up setting `restricted = false` in front50?